### PR TITLE
refactor: re-enable transaction weights on standalone and kintsugi

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: parachain/runtime/kintsugi/src/lib.rs [hunk 1]
```diff
@@ -284,7 +284,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 0;  // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    pub const TransactionByteFee: Balance = 1;
```
Comment (sander2) [inline parachain/runtime/kintsugi/src/lib.rs:287]: Shouldn't we set it to a more realistic value? I'm not sure how to arrive at such a value though.,

Comment (gregdhill) [reply parachain/runtime/kintsugi/src/lib.rs:287]: This is what was set previously, would need to check the effects upstream.

Comment (sander2) [reply parachain/runtime/kintsugi/src/lib.rs:287]: It is just an additional cost per byte of the encoded call - basically to pay for decoding them

**Comment by gregdhill** [comment]
Closed in favour of https://github.com/interlay/interbtc/pull/447

**Pull request closed without merging**